python/mujinwebstackclient/webstackgraphclientutils.py

```diff
@@ -36,8 +36,17 @@ class GraphClientBase(object):
 
     _webclient = None # an instance of ControllerWebClientRaw
 
-    def __init__(self, webclient):
+    def __init__(self, webclient, zmqClient=None):
         self._webclient = webclient
+        self._webstackZmqClient = zmqClient
+
+    def _CallGraphAPIV2(self, query, variables=None, timeout=5.0):
+        payload = {
+            'path': '/api/v2/graphql',
+            'query': query,
+            'variables': variables,
+        }
+        return self._webstackZmqClient.SendCommand(payload, timeout=timeout)['data']
```

Comment:
```suggestion
        return self._webstackZmqClient.SendCommand(payload, checkpreempt=False, timeout=timeout)['data']
```

Note: checkpreempt=False is from simulator.mujin.supervisordclient, pretty not sure